### PR TITLE
Render cluster-lifecycle chart directly

### PR DESCRIPTION
**Pull request by JakobGray**
Issue: https://github.com/stolostron/backlog/issues/25587

Moves source-of-truth for MCH cluster lifecycle templates from https://github.com/stolostron/multiclusterhub-repo/tree/main/multiclusterhub/charts/cluster-lifecycle/templates and CRDs from https://github.com/stolostron/hub-crds/tree/main/crds/cluster-lifecycle-chart to this repo.

Signed-off-by: Jakob Gray <20209054+JakobGray@users.noreply.github.com>

### CODE REVIEW

File: pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# (C) Copyright IBM Corporation 2020 All Rights Reserved
+# US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+###############################################################################
+# Copyright (c) 2020 Red Hat, Inc
+
+apiVersion: v1
+description: A helm chart for deploying the cluster lifecycle
+name: cluster-lifecycle
+appVersion: "2.5"
+version: 2.5.0
```
Comment (zhiweiyin318) [inline pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml:14]: Is there some rules about the version for the addon chart?  should we consider to keep the same as the acm version? we always forget to update it in each release.

File: pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright Contributors to the Open Cluster Management project
+
+###############################################################################
+# Licensed Materials - Property of IBM
+# (C) Copyright IBM Corporation 2020 All Rights Reserved
+# US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
```
Comment (zhiweiyin318) [inline pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml:6]: I think the IBM license is not needed anymore.

**Comment by zhiweiyin318** [comment]
some tiny comment. 
looks good to me.

File: pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml (hunk 1 above)
Comment (JakobGray) [reply pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml:14]: That version field isn't going to have any effect on the resources deployed. It isn't something we need worry to keep up to date.

File: pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml (hunk 2 above)
Comment (JakobGray) [reply pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml]: Removed

**Pull request merged**